### DISCUSSION TITLE
[Feature] On demand backup selected photos

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3427] [Feature Request]
### The feature

Very appreciated would be an option to backup selected photos in the same way as in Google Photos.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Can you clarify what you mean?

**Reply by waclaw66** [reply]
I don't want to have automatic backup enabled, but I need to backup/upload certain photos to the cloud like this in Google Photos...

![obrazek](https://github.com/immich-app/immich/assets/15554561/f2b0b866-63ba-4603-b487-d4c801a34785)

**Comment by waclaw66** [comment]
Seems a dup of https://github.com/immich-app/immich/discussions/1684
